test/mochitest/browser_dbg-debug-line.js

```diff
@@ -0,0 +1,38 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// Test ensures zombie debug lines do not persist
+// https://github.com/firefox-devtools/debugger/issues/7755
+add_task(async function() {
+  const dbg = await initDebugger("doc-debug-line.html");
+
+  // Ensure test files load
+  await waitForSources(dbg, "debug-line-1.js", "debug-line-2.js");
+
+  // Add breakpoint to debug-line-2
+  await addBreakpoint(dbg, "debug-line-2.js", 5);
+
+  // Reload
+  invokeInTab("doThing");
+
+  // Ensure we're paused
+  await waitForPaused(dbg);
+
+  // Click the call stack to get to debugger-line-1
+  const otherLocation = findAllElementsWithSelector(dbg, ".frame")[1];
```

Comment:
we should have a frame element in the selectors index. 
Also, it would be nice to use the `nth-child` trick we use in other selectors

Reply:
Fair enough; I think `:not(.selected)` could work too.